internal/istiooperator/istio-operator-light.yaml

```diff
@@ -186,6 +186,7 @@ spec:
       validationURL: ""
     cni:
       repair:
+        repairPods: false
```

Comment:
The repair pods need to be false, and delete true until https://github.com/istio/istio/commit/678762f4614c86e316612ffa8ac9d0b42acfce35 becomes available in a release. Without it, repair fails